packages/sit-onyx/src/components/OnyxMiniSearch/OnyxMiniSearch.vue

```diff
@@ -61,7 +61,8 @@ defineExpose({
       class="onyx-mini-search__clear"
       :aria-label="t('select.clearSearch')"
       tabindex="-1"
-      @mousedown.prevent="emit('clear')"
+      @mousedown.prevent
+      @click="emit('clear')"
```

Comment:
Isn't this the same as before?

Reply:
This is the bug fix for `fix(OnyxSelect): fix bug where OnyxSelect closed when clicking the clear button`
I added a comment explaining the issue.